CHANGELOG.md

```diff
@@ -44,18 +44,40 @@ should change the heading of the (upcoming) version to include a major version b
 - Fix [#3608](https://github.com/rjsf-team/react-jsonschema-form/issues/3608) by ensuring the root field is always wrapped in Form.Item
 - Fix [#3594](https://github.com/rjsf-team/react-jsonschema-form/issues/3594) by removing the duplicate title for `SelectWidget` and description for `CheckboxWidget`
 
+## @rjsf/bootstrap-4
```

Comment:
Since we just released 5.7.2, would you be willing to move all of these into a new 5.8.0 section?

Reply:
I moved my notes into a new section